packages/i18n/src/lang/fr.json

```diff
@@ -300,7 +300,7 @@
     "description": "Message displayed to user after attempting to update a candidate with an unexpected status."
   },
   "AvBSV+": {
-    "defaultMessage": "Oh non... ",
+    "defaultMessage": "Oh non...",
```

Comment:
Just a question, have we tried `&hellip;` instead of `...`?

Reply:
- [11d980f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12536/commits/11d980ff41d7de58eb45838005e5a0f4cbc8f457)
- [679cb82](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12536/commits/679cb825b0e38a489475f7a8ced3f3c1b39f6a95)